package.json

```diff
@@ -48,6 +48,7 @@
     "@babel/preset-typescript": "7.12.1",
     "@babel/runtime": "7.12.5",
     "@cypress/skip-test": "2.5.0",
+    "@guardian/libs": "1.6.1",
```

Comment:
you'll need to add this as a peerDep too, to make sure it's available when the CMP is installed. prob a good idea to add that as `^1.6.1`